src/backend/app.js

```diff
@@ -21,10 +21,76 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+const knex = require("./database");
+
+router.get("/all-meals", async (request, response) => {
+  try {
+    const allMeals = await knex("Meal").select("*").orderBy('id', 'desc');
+    response.json(allMeals);
+  }
+  catch (error) {
+    throw error;
+  }
+});
+
+router.get("/future-meals", async (request, response) => {
+  try {
+    const futureMeals = await knex("Meal").select("*").where("created_date", '>=', '2023-01-01');
```

Comment:
You can use datetime for the current date: `const currentDate = new Date().toISOString().slice(0, 10);`